<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceWidget</class>
 <widget class="QWidget" name="ReferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="frameWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_Publication">
        <property name="text">
         <string>The publication to cite:</string>
        </property>
        <property name="buddy">
         <cstring>publicationEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PublicationEdit" name="publicationEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_CiteKey">
       <property name="text">
        <string>Citekey:</string>
       </property>
       <property name="buddy">
        <cstring>citeKeyEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="StringEdit" name="citeKeyEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Pages">
       <property name="text">
        <string>Pages:</string>
       </property>
       <property name="buddy">
        <cstring>pagesEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="StringEdit" name="pagesEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rating:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KRatingWidget" name="editRating"/>
     </item>
     <item row="2" column="1">
      <widget class="ChapterEdit" name="chapterEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Chapter">
       <property name="toolTip">
        <string>Part of the publication. A chapter or quote</string>
       </property>
       <property name="text">
        <string>Part:</string>
       </property>
       <property name="buddy">
        <cstring>chapterEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>275</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRatingWidget</class>
   <extends>QFrame</extends>
   <header>kratingwidget.h</header>
  </customwidget>
  <customwidget>
   <class>StringEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/stringedit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>PublicationEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/publicationedit.h</header>
  </customwidget>
  <customwidget>
   <class>ChapterEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/chapteredit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>citeKeyEdit</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_CiteKey</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pagesEdit</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_Pages</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chapterEdit</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_Chapter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showPublicationList()</slot>
  <slot>createReference()</slot>
  <slot>removeReference()</slot>
 </slots>
</ui>
